comp/core/status/statusimpl/status.go

```diff
@@ -307,9 +310,10 @@ func (s *statusImplementation) GetStatusBySection(section string, format string,
 			return []byte{}, nil
 		}
 	default:
-		providers, ok := s.sortedProvidersBySection[section]
+		providers, ok := s.sortedProvidersBySection[strings.ToLower(section)]
 		if !ok {
-			return nil, fmt.Errorf("unknown status section '%s'", section)
+			availableSections := strings.Join(s.sortedSectionNames, "\n- ")
```

Comment:
Should we `Join` using `", "` in the case of an error when requesting a particular section in `json` I'm not sure using new lines is going to be readable